controllers/api.js

```diff
@@ -15,6 +15,12 @@ redis.monitor(function (err, monitor) {
   });
 });
 
+/****** START SET FEES FROM CONFIG AT STARTUP ******/
+/** GLOBALS */
+global.forwardFee = config.defaultForwardReserveFee;
```

Comment:
ok, if you intend to change them at runtime it makes sense to have them as global, instead of using `config.bla`.
but how often is this gonna happen? if you have some plugin code that edit those values, why not save them to config and request a restart of lndhub?

Reply:
yes . true . this was 2 fold, both fixing the asked for fee settings functionality and at same time be able demonstrate LndHub-Admin extension example, where also login and then set new fees if wanted / needed can be had. It coincided and makes for a great demo, it was the quickest at hand for demo using the new feeSetting.

i find it handy be able set fees at runtime YMMV

Reply:
also the umbrel ++ ppl need a UI setting fees can not easily edit dockerimages pulled from net and proficiency in commandline might have developed for them yet !

Reply:
but You say UI should write new config and the manually restart ?

can be problematic with different distros of node environments with paths and permissions and what not ?